index.html

```diff
@@ -1626,7 +1691,9 @@
             if (style.indexOf(';') == -1) {
                 style = style + ';';
             }
-
+            else if ( style[style.length-1] != ';'){
```

Comment:
How is this different from the if condition above? Can you write a comment?
Also, beautify!